src/pages/inspect/tx/ui/tx-view-component/utils/cn.tsx

```diff
@@ -0,0 +1,6 @@
+import { type ClassValue, clsx } from 'clsx';
+import { twMerge } from 'tailwind-merge';
+
+export function cn(...inputs: ClassValue[]): string {
+  return twMerge(clsx(inputs));
+}
```

Comment:
question: in what situation does it help us? It seems like `cn()` function is used only on basic HTML elements without contradicting classes, so replacing it with `clsx` wouldn't change anything, right?

Reply:
I noticed that here it behaves differently (notice the text color):

```
cn(
            'flex items-center justify-center text-text-secondary text-sm font-bold rounded-sm flex-1 h-[38px]',
            value === option.value && 'bg-secondary-main text-text-primary',
          )
```

Made a small recording to demonstrate:

https://github.com/user-attachments/assets/34d754a2-73f4-4aeb-b197-fcf8bda5e078

Reply:
Ah, i see, that's a real case. What about this change?

```
cn(
  'flex items-center justify-center text-sm font-bold rounded-sm flex-1 h-[38px]',
  value === option.value ? 'bg-secondary-main text-text-primary' : 'text-text-secondary',
)
```

it requires more control over the classes use but at least saves [7.5 kB](https://bundlephobia.com/package/tailwind-merge@3.0.1)